argus/backend/error_handlers.py

```diff
@@ -12,14 +15,24 @@ class DataValidationError(APIException):
     pass
 
 def handle_api_exception(exception: Exception):
+    trace_id = base64.encodebytes(sha256(os.urandom(64)).digest()).decode(encoding="utf-8").strip()
     if issubclass(exception.__class__, APIException):
+        LOGGER.info("TraceId: %s", trace_id)
```

Comment:
maybe prefix all the info messages with the traceId? so it could be easily grepped

Reply:
Currently I'm adding it to top and bottom of the dump, which I think should be enough. WDYT?

Reply:
but if you just grep, then you will get only these 2 lines. But from the other hand, all python trace lines won't be shown too as they won't include it also..

Reply:
Maybe including just one at the top and using `grep -A 15` would be better then? (using two in this case you'll get extra lines from the end marker